packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -537,13 +537,20 @@ class TextSelectionOverlay {
     _handleSelectionHandleChanged(newSelection, isEnd: false);
   }
 
+  TextSelection? _lastSelection;
+
   void _handleSelectionHandleChanged(TextSelection newSelection, {required bool isEnd}) {
     final TextPosition textPosition = isEnd ? newSelection.extent : newSelection.base;
     selectionDelegate.userUpdateTextEditingValue(
       _value.copyWith(selection: newSelection),
       SelectionChangedCause.drag,
     );
     selectionDelegate.bringIntoView(textPosition);
+
```

Comment:
Are these changes still needed?

Reply:
these changes are needed to trigger feedback when selection changes due to either handle being dragged, I'm not sure how I would re-implement this in `SelectionOverlay`.